src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.alive = [];
```

Comment:
The `alive` property should be a static property on the `Animal` class, not an instance property. Move it to after the class definition and initialize it as an empty array.